src/test/java/com/hocs/test/pages/to/TOProgressCase.java

```diff
@@ -178,6 +199,28 @@ private void moveCaseFromTriageToStopList() {
         clickTheButton("Confirm");
     }
 
+    public void moveCaseFromTriageToCCH() {
+        String currentBusinessArea = sessionVariableCalled("businessArea");
+        openOrCloseAccordionSection("Case Details");
+        clickTheLink("Change business area");
+        waitABit(500);
+        if (currentBusinessArea.equalsIgnoreCase("UKVI")) {
+            List<WebElementFacade> listOfNonCheckedBusinessAreas = findAll("//input[not(@checked)]/following-sibling::label");
+            Random rand = new Random();
+            WebElementFacade randomBusinessArea = listOfNonCheckedBusinessAreas.get(rand.nextInt(listOfNonCheckedBusinessAreas.size()));
+            listOfNonCheckedBusinessAreas.remove(randomBusinessArea);
+            randomBusinessArea = listOfNonCheckedBusinessAreas.get(rand.nextInt(listOfNonCheckedBusinessAreas.size()));
+            safeClickOn(randomBusinessArea);
+            safeClickOn(finishButton);
+            caseView.clickAllocateToMeLink();
+            openOrCloseAccordionSection("Case Details");
+            clickTheLink("Change business area");
+            waitABit(500);
+        }
```

Comment:
I don't thin this is needed. If we add the line I suggested above then we can rely on the business area already being one that is different from UKVI